draft-bmw-tls-pake13.md

```diff
@@ -68,6 +68,16 @@ Enabling TLS to address this use case effectively requires the TLS
 handshake to execute a password-authenticated key establishment
 (PAKE) protocol. This document describes a TLS extension `pake`
 that can carry data necessary to execute a PAKE.
+{{Section 9.2 of !TLS13=RFC8446}} specifies that a valid Client
+Hello must include either a `pre_shared_key` extension or both
```

Comment:
Super nitpicky nitpick: I have never understood whether you are supposed to say "Client Hello" or "ClientHello" in prose. RFC 8446 uses "Client Hello" in section headers, but all the text says "ClientHello". I guess that would suggest this should say ClientHello. 🤷